src/System.Private.CoreLib/src/System/Reflection/Emit/SignatureHelper.cs

```diff
@@ -376,7 +378,7 @@ private void AddOneArgTypeHelperWorker(Type clsArgument, bool lastWasGenericInst
                 }
                 else
                 {
-                    tkType = m_module.GetTypeToken(clsArgument);
+                    tkType = m_module!.GetTypeToken(clsArgument); //TODO-NULLABLE possible compiler bug
```

Comment:
What's the bug?  Please include the link to the issue in the Roslyn repo.

Reply:
seem its not a bug